source/ContextualPopup.js

```diff
@@ -542,6 +542,10 @@
 			this.inherited(arguments);
 			this.alterDirection();
 			this.showHideScrim(this.showing);
+
+			if(this.spotActivatorOnClose) {
+				enyo.Spotlight.spot(this.activator);
```

Comment:
Good catch Kunmyon. Added check for `this.showing` so spotlight isn't sent unless the popup was dismissed.